packages/ui/src/components/utils/popover/index.tsx

```diff
@@ -52,25 +52,34 @@ export const Popover = forwardRef<HTMLDivElement, PopoverProps>(
     ): ReactElement {
         const [popper, setPopper] = useState<HTMLDivElement | null>(null);
 
-        useEffect(() => {
-            if (!popper || !ref) return;
-            if (typeof ref === "function") ref(popper);
-            else ref.current = popper;
-        }, [popper, ref]);
+        const { styles, attributes, update, forceUpdate } = usePopper(
+            anchor,
+            popper,
+            {
+                placement,
+                modifiers: [
+                    {
+                        name: "offset",
+                        options: { offset },
+                    },
+                ],
+            }
+        );
+        console.log(anchor, styles, attributes, update, forceUpdate);
```

Comment:
`console.log` here.

Reply:
Addressed in 1a11900364650c00d0b8a8df8bcfa484ef6dfaaf